cpp/daal/include/services/internal/execution_context.h

```diff
@@ -127,10 +127,12 @@ class SyclExecutionContext : public ExecutionContext
      *  are performed on the device associated with the queue
      *  \param[in] deviceQueue SYCL* queue object to the device that is selected to perform computations
      */
-    SyclExecutionContext(const ::sycl::queue & deviceQueue) : ExecutionContext(createContext(deviceQueue), !deviceQueue.get_device().is_cpu()) {}
+    SyclExecutionContext(const ::sycl::queue & deviceQueue, const bool fromPython = false)
+        : ExecutionContext(createContext(deviceQueue, fromPython), !deviceQueue.get_device().is_cpu())
+    {}
 
 private:
-    static daal::services::internal::sycl::ExecutionContextIface * createContext(const ::sycl::queue & queue)
+    static daal::services::internal::sycl::ExecutionContextIface * createContext(const ::sycl::queue & queue, const bool fromPython = false)
```

Comment:
We already deprecated this function, I really don't see any reason for such update.
We can leave the signature as is, no updates its API required.